### PR TITLE
Add rules for contractor credential objects

### PR DESCRIPTION
**Pull request by WadeBarnes**
- Details can be found in https://github.com/bcgov/DITP/issues/95

### CODE REVIEW

File: configurations/CAndy/dev/allowed-did.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+registered_did,details
+<CDT DID TBD>,Cyber Security and Digital Trust
+<CSB DID TBD>,Court Services Branch
```
Comment (WadeBarnes) [inline configurations/CAndy/dev/allowed-did.csv:3]: CDT and CSB DIDs to be defined as part of https://github.com/bcgov/DITP/issues/95

File: configurations/CAndy/dev/allowed-schema.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+author_did,schema_name,version,details
+<CDT DID TBD>,contractor-credential,1.0,https://github.com/bcgov/digital-trust-toolkit/blob/contractor-credential/docs/governance/employment/contractor-credential/governance.md#261-schema-definition
```
Comment (WadeBarnes) [inline configurations/CAndy/dev/allowed-schema.csv:2]: Question for @Gavinok or @esune,
Does the details field support arbitrary text, such as the reference URL I've specified here?

Comment (esune) [reply configurations/CAndy/dev/allowed-schema.csv]: Yes, the details field is free text - I forget how long it can be though.

**Comment by esune** [comment]
Should we create generic wildcards for `dev`, without specifying specific DIDs? This would help us with onboarding any new development team without having to vet them first, since this environment is likely going to be the first proving ground.

**Comment by WadeBarnes** [comment]
When onboarding a new agent (team) with the endorser we have to register the agent with the endorser in one of two ways since we don't, and should never, allow open endorsement to all:
- configure the per-connection settings for auto-endorsement.
- register a set of rules for the new agent.

The first method is hidden and includes some manual/scripted steps.  The second method is openly visible, descriptive, and has the added benefit of the history in git.

My preference is to onboard teams by registering their DIDs, and then we can decide to provide them with open (wildcard) or more restricted access to write schemas and cred defs.

I've purposely added some restrictions to both CDT and CSB rules.  CDT, simply because this is the first time we're creating a DID for ourselves and I'd like to record our use cases as we evolve them.  We have full control so it's easy for us to update our own rules.  For this use case we've defined governance rules defining the rules and responsibilities of each party, these rules reflect those defined by the governance documentation and explicitly reflect the fact that CDT is responsible for issuing the schema and CSB is responsible for issuing the cred-def based on that schema.

**Pull request merged**